charts/sourcegraph-appliance/values.yaml

```diff
@@ -0,0 +1,109 @@
+# Default values for sourcegraph-appliance.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: localhost:5001
```

Comment:
In that case, how are you pushing to the local registry such that kind can access it?

Reply:
💡 ah, sorry for not explaining further - my docker-tag approach only works because imagePullPolicy is set to IfNotPresent, **and** my local kubernetes shares the docker daemon that runs it. I'm using docker desktop's kubernetes, specifically, and this works out of the box.

This is an alternative to running a registry in KinD, as you document in the PR description - I should have been clearer about that!

I would recommend sharing the docker daemon with the inner kubernetes' container runtime to simplify this sort of thing, but I'm not sure how to configure this in KinD 🤔

Reply:
But I think the note still stands, we should use a default that is sensible for users, and developers can override it in a values file / `--set`.

Reply:
Definitely agree that this is a poor default, and that `index.docker.io/sourcegraph` is the correct default.